java/src/org/openqa/selenium/grid/node/docker/DockerSessionFactory.java

```diff
@@ -116,7 +116,8 @@ public DockerSessionFactory(
       Image videoImage,
       DockerAssetsPath assetsPath,
       String networkName,
-      boolean runningInDocker) {
+      boolean runningInDocker,
+      SlotMatcher slotMatcher) {
```

Comment:
We should pass a `Predicate` here, in the same way it was done at the `LocalNodeFactory`.

Reply:
updated